images/runtime/php-fpm/7.2/base.stretch.Dockerfile

```diff
@@ -89,31 +89,13 @@ RUN { \
                 echo 'opcache.enable_cli=1'; \
     } > /usr/local/etc/php/conf.d/opcache-recommended.ini
 
+# NOTE: zend_extension=opcache is already configured via docker-php-ext-install, above
 RUN { \
                 echo 'error_log=/var/log/apache2/php-error.log'; \
                 echo 'display_errors=Off'; \
                 echo 'log_errors=On'; \
                 echo 'display_startup_errors=Off'; \
                 echo 'date.timezone=UTC'; \
-                echo 'zend_extension=opcache'; \
-    } > /usr/local/etc/php/conf.d/php.ini
-
-RUN { \
```

Comment:
What's the reason for removing these two `RUN` commands in the PHP FPM images?

Reply:
These are duplicate commands. Note starting on line 83: https://github.com/microsoft/Oryx/pull/1452/files#diff-18d17deb4df0320c56cc64ce0b0b02f2199b5b405f58e0d7e30e024d2e79a7d6R83

Reply:
Ah, makes sense, I didn't expand the file upwards 😄 It does appear that the last `echo` of the second `RUN` command was removed, do you know if that was intentional?

```
echo 'zend_extension=opcache'; \
```

Reply:
Yup, this will remove the PHP warning message mentioned in https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1554458